src/features/Console.ts

```diff
@@ -220,5 +262,25 @@ export class ConsoleFeature implements IFeature {
         this.languageClient.onRequest(
             ShowInputPromptRequestType,
             (promptDetails) => showInputPrompt(promptDetails, this.languageClient));
+
+        // Set up status bar alerts for when PowerShell is executing a script
+        this.languageClient.onNotification(
+            ExecutionStatusChangedNotificationType,
+            (executionStatusDetails) => {
+                switch (executionStatusDetails.executionStatus) {
+                    // If execution has changed to running, make a notification
+                    case ExecutionStatus.Running:
+                        this.resolveStatusBarPromise = showStatusMessage("Executing PowerShell...");
+                        break;
+
+                    // If the execution has stopped, destroy the previous notification
+                    case ExecutionStatus.Completed:
+                    case ExecutionStatus.Aborted:
+                    case ExecutionStatus.Failed:
+                        this.resolveStatusBarPromise();
```

Comment:
if for whatever reason... vscode powershell doesn't get the ExecutionStatus.Running message but it gets the ExecutionStatus.Completed message... will this line throw an error if `this.resolveStatusBarPromise` takes the default value?